honeybee_radiance_command/gensky.py

```diff
@@ -0,0 +1,221 @@
+                GST -4, IST -5.5, JST -9, NZST -12, YDT +8, PDT +7, MDT +6, CDT +5,
+                EDT +4, BST -1, CEST -2, EEST -3, ADT -4, GDT -5, IDT -6.5, JDT -10,
+                NZDT -13
+            solar_time: A boolean to use local solar time. If set to True then the time
+                is preceded by '+' sign and local solar time is used instead of local
+                standard time.
+            options: Command options. It will be set to Radiance default values if not
+                provided by user.
+            output: File path to the output file (Default: None).
+        """
+        Command.__init__(self, output=output)
+        self.options = options
+        if ang:
+            self.options = ang
+            self._ang = ang
+            self._month = None
+            self._day = None
+            self._time = None
+            self._time_zone = None
+            self._solar_time = None
+        else:
+            self._ang = None
+            self._month = month
+            self._day = day
+            self._time = time
+            self._time_zone = time_zone
+            self._solar_time = solar_time
+
+    @property
+    def options(self):
+        """Rpict options."""
+        return self._options
+
+    @options.setter
+    def options(self, value):
+        if not value:
+            value = GenskyOptions()
+
+        if not isinstance(value, GenskyOptions):
+            raise ValueError('Expected GenskyOptions not {}'.format(type(value)))
+
+        self._options = value
+
+    @property
+    def ang(self):
+        """Angles for altitude and azimuth."""
+        return self._ang
+
+    @ang.setter
+    def ang(self, value):
+        self._ang.value = value
+
+    @property
+    def month(self):
+        """Month."""
+        return self._month
+
+    @month.setter
+    def month(self, value):
+        if value and isinstance(value, int) and value in range(1, 13):
+            self._month = value
+        else:
+            raise ValueError('The value of month needs to be an integer between'
+                             ' 1 and 12 inclusive. Instead got %.' % (value))
+
+    @property
+    def day(self):
+        """Day."""
+        return self._day
+
+    @day.setter
+    def day(self, value):
+        if value and isinstance(value, int) and value in range(1, 32):
+            self._day = value
+        else:
+            raise ValueError('The value of day needs to be an integer between'
+                             ' 1 and 31 inclusive. Instead got %.' % (value))
+
+    @property
+    def time(self):
+        """time."""
+        return self._time
+
+    @time.setter
+    def time(self, value):
+        if value and isinstance(value, float, int):
+            hour, minute = str(float(value)).split('.')
+            if 1 <= int(hour) <= 24 and 0 <= int(minute) <= 59:
+                self._time = value
+            else:
```

Comment:
This validation doesn't look correct. Here is the documentation: `By default, the time is interpreted as local standard time on a 24-hour clock. The time value may be given either as decimal hours, or using a colon to separate hours and minutes.`

Based on the documentation the values can either be `9.5` or `9:30` and they should both work.